parm/config/config.fcst

```diff
@@ -133,13 +172,45 @@ elif [ $imp_physics -eq 11 ]; then # GFDL
     export hord_xx_nh_nonmono=5
     export vtdm4_nh_nonmono=0.02
     export nord=2
-    export dddmp=0.1
     export d4_bg=0.12
+    export dddmp=0.1
 
 else
     echo "Unknown microphysics option, ABORT!"
 
 fi
+
+#coupling settings
+export FRAC_GRID=".true."
+export cplmode="nems_frac"
+export psm_bc="1"
+
+export min_lakeice="0.15"
+export min_seaice="1e-11"
+
+export FSICL="99999"
+
+# Stochastic physics parameters
+# nam_stochy section
+export SEEDLET="10"
+export DO_SKEB="YES"
+export DO_SHUM="YES"
+export DO_SPPT="YES"
+
+export SKEB=0.6,-999,-999,-999,-999
+export iseed_skeb=$ISEED_SKEB
+export SKEB_TAU=2.16E4,1.728E5,2.592E6,7.776E6,3.1536E7
+export SKEB_LSCALE=500.E3,1000.E3,2000.E3,2000.E3,2000.E3
+export SKEBNORM=1
+
+export SPPT=0.8,0.4,0.2,0.08,0.04
+export iseed_sppt=$ISEED_SPPT
+export SPPT_TAU=2.16E4,2.592E5,2.592E6,7.776E6,3.1536E7
+export SPPT_LSCALE=500.E3,1000.E3,2000.E3,2000.E3,2000.E3
+export sppt_logit=".true."
+export sppt_sfclimit=".true."
+export use_zmtnblck=".true."
```

Comment:
Aren't these relevant to ensemble forecasts?
So, shouldn't they be in `config.efcs`?

Reply:
config.efcs has stochastic physics settings already, see these lines: https://github.com/NOAA-EMC/global-workflow/blob/develop/parm/config/config.efcs#L28:L45

Reply:
Will remove block.